Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -395,7 +395,7 @@ private object EmptyCString: CValues<ByteVar>() {
     override val align get() = 1
 
     private val placement =
-            interpretCPointer<ByteVar>(nativeMemUtils.alloc(1, 1).rawPtr)!!.also {
+            interpretCPointer<ByteVar>(nativeMemUtils.allocRaw(1, 1))!!.also {
```

Comment:
(Not related to this PR but) this one probably leaks too. Although this happens once per classloader, so might be ok.

Reply:
Yes, it is, but it's only 8 byte (after padding), so probably doesn't matter.